packages/http-client-csharp/generator/Microsoft.Generator.CSharp/src/Expressions/PositionalParameterReferenceExpression.cs

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Microsoft.Generator.CSharp.Expressions
+{
+    public sealed record PositionalParameterReferenceExpression(string ParameterName, ValueExpression ParameterValue) : ValueExpression
```

Comment:
I think this should go with ParameterReference wherever that thing goes because this is basically the same thing

Reply:
Opened a tracking item for this as its not directly related to refactoring namespaces https://github.com/microsoft/typespec/issues/3472